cli/bin/exec.js

```diff
@@ -2,6 +2,14 @@ const path = require("path");
 const { execSync } = require("child_process");
 const fs = require("fs");
 
+function exec(command, execOpts) {
+  try {
+    return execSync(command, execOpts);
+  } catch (err) {
+    process.exit(err.status);
```

Comment:
Never force exit in node.

Reply:
Why?

Reply:
I found good information in https://nodejs.org/api/process.html#processexitcode.

Reply:
@spotandjake Looks like you can just set `process.exitCode` and that code will be used when the process exits.

Reply:
Yeah. You want to submit another PR once I merge this?